src/applications/vaos/vaos-entry.jsx

```diff
@@ -0,0 +1,11 @@
+import '../../platform/polyfills';
+
```

Comment:
We probably also want to create a sass file for this app, since the react page template will add a tag for that automatically.